tests/foreman/api/test_webhook.py

```diff
@@ -170,3 +170,27 @@ def test_positive_event_triggered(self, module_org, target_sat):
             shell.send('foreman-tail')
             repo.sync()
             assert_event_triggered(shell, hook.event)
+
+    @pytest.mark.tier1
+    @pytest.mark.parametrize('setting_update', ['safemode_render=False'], indirect=True)
+    def test_negative_event_task_failed(self, module_org, target_sat, setting_update):
+        """Create a webhook with unreachable targete and assert the associated task
+        failed
+
+        :id: d4a49556-9413-46e8-bcb5-7afd0184bdb2
+
+        :expectedresults: Deliver webhook task fails
+
+        :CaseImportance: High
+        """
+        hook = entities.Webhooks(
```

Comment:
```suggestion
        hook = target_sat.api.Webhooks(
```
Same suggestion for other occurrences of `entities`.